x/dclauth/handler_test.go

```diff
@@ -192,6 +192,17 @@ func TestHandler_CreateAccount_ThreeApprovalsAreNeeded(t *testing.T) {
 	require.False(t, setup.Keeper.IsRevokedAccountPresent(setup.Ctx, address))
 }
 
+func TestHandler_NeededApprovalCountForManyTrustees(t *testing.T) {
```

Comment:
Can we please have a similar test for Vendor Account approvals (1/3 of Trustees)?

Reply:
Done